### DISCUSSION TITLE
GET changes in LAB module

### DISCUSSION THREAD

**Discussion by james-eda** [cdisc-org/DataExchange-DatasetJson-API#12] [User Stories]
As an SDTM programmer, I want to inherit all the latest units and ranges most recently submitted to local labs module of the EDC.

**Comment by JozefAerts** [comment]
Can there be a way to do such things in a generic way? I.e. that a variable (or list of variables) is used as a parameter.

**Reply by mhungria** [reply]
This user story could be combined with user story #5 (Get datasets changed after a given date).

**Comment by james-eda** [comment]
[like]  James Joseph reacted to your message:
________________________________

**Comment by swhume** [comment]
As I noted on Slack, our user stories seem to be addressing 2 different, but useful, needs. One is a simple interface for dataset exchange only. This is the basic API anyone doing data exchange with Dataset-JSON should implement. Basically, just moving datasets and none of the more sophisticated processing.  The second, provides the means for programmers to query the datasets to get very specific data out of the datasets. This second set of use cases might be more generically implemented in GraphQL. So, maybe we end up with 2 APIs as they address different needs and there's value in keeping the base data exchange API very simple to implement and use. Thoughts?

**Comment by james-eda** [comment]
[like]  James Joseph reacted to your message:
________________________________
